boogiestats/boogie_ui/views.py

```diff
@@ -147,6 +147,12 @@ def get_context_data(self, **kwargs):
             ("10+", "min-plays-10"),
             ("15+", "min-plays-15"),
             ("20+", "min-plays-20"),
+            ("25+", "min-plays-25"),
```

Comment:
You also have to update `mappings` in L81. I'd rather have that list extracted to the global namespace and used both there and here to have SSOT.

Reply:
You're right, I didn't even notice there is another place where it should be changed.
I created `CALENDAR_VALUES` so it should be easier now to change/add it if needed.